packages/vat-data/src/vat-data-bindings.js

```diff
@@ -0,0 +1,107 @@
+/* global globalThis */
+
+import { assert } from '@agoric/assert';
+import {
+  M,
+  makeScalarMapStore,
+  makeScalarWeakMapStore,
+  makeScalarSetStore,
+  makeScalarWeakSetStore,
+  provide,
+} from '@agoric/store';
+
+export {
+  M,
+  makeScalarMapStore,
+  makeScalarWeakMapStore,
+  makeScalarSetStore,
+  makeScalarWeakSetStore,
+};
+
+/** @type {import('./types').VatData} */
+let VatDataGlobal;
+if ('VatData' in globalThis) {
+  assert(globalThis.VatData, 'VatData defined in global as null or undefined');
+  VatDataGlobal = globalThis.VatData;
+} else {
+  // XXX this module has been known to get imported (transitively) in cases that
+  // never use it so we make a version that will satisfy module resolution but
+  // fail at runtime.
+  const unvailable = () => assert.fail('VatData unavailable');
+  VatDataGlobal = {
```

Comment:
I don't get what's going on here.   Why do these names have to appear in this list?  Why are there no others?

Reply:
@Chris-Hibbert , if @turadg 's explanation is an adequate answer, please resolve this conversation. Thanks.